client/components/jetpack/backup-getting-started/consts.js

```diff
@@ -0,0 +1,2 @@
+export const GETTING_STARTED_WITH_JETPACK_BACKUP_VIDEO_URL =
```

Comment:
If you're comfortable with TypeScript, favour using `ts` and `tsx` extensions.

Reply:
I am comfortable with TS indeed, I just thought we stick to JS in this project, will adjust!

Reply:
I believe we now want to use TS by default in Calypso (at least for everything Jetpack), but definitely let me know if you've seen something different.

Reply:
No, it's just that I didn't know about that, thanks for clarifying